### ISSUE TITLE
Staging Review finding: Disabled Continue button 

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#111492]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Unified Appointment Experience
**Product:** Appointments
**Feature:** Community Care Direct Scheduling

## Findings details
**VA.gov Experience Standard - issue:** [User can complete necessary flows.](https://design.va.gov/about/experience-standards/#user-can-complete-necessary-flows)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

On the "How to Schedule" page when viewing the facility "Spokane VA Clinic", the Continue button is greyed-out. They cannot complete that action via that flow, they need to navigate with the link or call.

This button also does not have sufficient contrast between the text and its background.

<details><summary>Link, screenshot or steps to recreate</summary>

![Image](https://github.com/user-attachments/assets/d266463c-3ce4-4795-b60b-a67f18f34d92)

</details>

## Recommended action

If the user cannot proceed, this button should not be shown. The messaging could also be clearer on why they cannot continue.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG21/#info-and-relationships)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
